src/AppBundle/Resources/views/menu.html.twig

```diff
@@ -83,7 +83,7 @@
       {% set deliveryAddress = app.session.get('deliveryAddress') %}
 
       {% if deliveryAddress is empty and cart is not empty %}
-      <a href="{{ path('restaurant', {'id': cart.restaurantId}) }}" class="btn btn-default navbar-btn navbar-right">
+      <a href="{{ path('restaurant', {'id': cart.restaurant.id}) }}" class="btn btn-default navbar-btn navbar-right">
```

Comment:
Causes an `Internal Server Error` when there is already a cart in session. 
We will need to delete sessions when deploying this.

Reply:
I try to keep the old carts valid - looking into this.

If no solution I think it is ok to remove all in-session carts.